package/src/components/AttachmentPicker/AttachmentPicker.tsx

```diff
@@ -50,17 +73,25 @@ const AttachmentImage: React.FC<AttachmentImageProps> = (props) => {
     numberOfAttachmentPickerImageColumns,
     onPress,
     selected,
+    type,
     uri,
+    videoDuration,
   } = props;
   const {
     theme: {
       attachmentPicker: { image, imageOverlay },
-      colors: { overlay },
+      colors: { overlay, white },
     },
   } = useTheme();
 
   const size = vw(100) / (numberOfAttachmentPickerImageColumns || 3) - 2;
 
+  const duration = videoDuration
+    ? videoDuration / 3600 >= 1
+      ? dayjs.duration(videoDuration, 'second').format('HH:mm:ss')
```

Comment:
Don't use dayjs directly, we should use out i18n setup to do this. Check the other places where we have datetime stamp how we do it